mmrazor/models/quantizers/academic_quantizer.py

```diff
@@ -54,6 +55,9 @@ def prepare(self, model, graph_module):
             setattr(graph_module, attr_name, getattr(model, attr_name))
         fuse_custom_config = FuseCustomConfig().set_preserved_attributes(
             preserved_attributes)
+
+        self.sync_module_training_mode(graph_module, self.is_qat)
```

Comment:
1. whether `self.sync_module_training_mode` be just used to handle some special cases? 
2. if 1 is yes, can it be use its own judgement to execute without `is_qat`
3. if 2 is no, had better not use `is_qat`, beacause `fuse` and `preprare`  do not distinguish `is_qat`, and some ptq algorithms may be to train model, such as `Adaround`

Reply:
Successive conv bn layers may be in different modes. For example, bn layers in detectors are usually in eval mode during training. But the successive conv bn layers should be in the same mode during `_fuse_fx`. 
It seems that all modules can be set to training mode. I fixed it just now.